arch/generic/chunk_128bit_perm_idx_lut.h

```diff
@@ -0,0 +1,26 @@
+/* chunk_128bit_perm_idx_lut.h - shared SSSE3/NEON/LSX permutation idx lut for use with chunkmemset family of functions.
+ * For conditions of distribution and use, see copyright notice in zlib.h
+ */
+
+#ifndef CHUNK_128BIT_PERM_IDX_LUT_H_
+#define CHUNK_128BIT_PERM_IDX_LUT_H_
+
+#include "chunk_permute_table.h"
```

Comment:
This include line will cause chicken and egg kind of loop... In at least some cases `chunk_permute_table.h` has already been included before including this file.